packages/plugins-core/src/types/actualPlugin.ts

```diff
@@ -0,0 +1,521 @@
+import type { ReactElement } from 'react';
+
+import type { BasicModalProps } from '@actual-app/components';
+import type { Query, QueryBuilder } from '@actual-app/query';
+import type {
+  AccountEntity,
+  CategoryEntity,
+  CategoryGroupEntity,
+  PayeeEntity,
+  ScheduleEntity,
+} from '@actual-app/shared-types';
+import type { i18n } from 'i18next';
+
+export type SlotLocations =
+  | 'main-menu'
+  | 'more-menu'
+  | 'before-accounts'
+  | 'after-accounts'
```

Comment:
nit: maybe prefix with `sidebar` to make it clear where it goes?
```suggestion
  | 'sidebar-main-menu'
  | 'sidebar-more-menu'
  | 'sidebar-before-accounts'
  | 'sidebar-after-accounts'
```